legaldb/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Country(models.Model):
```

Comment:
I recommend using a third party library such as https://github.com/SmileyChris/django-countries for the Country model